id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/dto/CredentialIssueRequestDto.java

```diff
@@ -20,5 +20,6 @@ public class CredentialIssueRequestDto {
 	private String encryptionKey;
     private List<String> sharableAttributes;
     private Map<String,Object> additionalData;
+    private String requestId;

```

Comment:
CredentialIssueRequestDto is used by both v1 and v2 endpoint. Introducing new field in this DTO will change the request structure for v1 endpoint as well. Can we create separate DTO or extend from existing DTO for v2 endpoint?

Reply:
Changes done